training/src/layouts/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { styleForFooter } from './style';
+
+class Footer extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
```

Comment:
Why are we using <> and </> if we are returning single component from the component?